gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/vertx/view/thymeleaf/ThymeleafConfiguration.java

```diff
@@ -35,8 +40,11 @@ public class ThymeleafConfiguration {
     @Autowired
     private Vertx vertx;
 
+    @Value("${template.i18n:#{systemProperties['gravitee.home']}/templates/i18n}")
```

Comment:
We have already a `templates` section in the gravitee.yml file : 

```
# Mail templates
#templates:
#  path: ${gravitee.home}/templates
```

Not sure if we have to re-think that part 🤔

Reply:
OK, so even the form templates will be moved here. That make sens, specially for user that want to update default template . It will be possible for them to define a new custom default template without having to override templates for each domain. :+1:

Reply:
maybe forms instead of pages to be consistent with the UI :thinking:

Reply:
yes

Reply:
this refactor will be handled by https://github.com/gravitee-io/issues/issues/8114
I will FP the change to use the templates.path settings with i18n as subdirectory